src/lib/features/metrics/instance/instance-service.ts

```diff
@@ -114,7 +114,7 @@ export default class ClientInstanceService {
         this.seenClients[this.clientKey(data)] = data;
     }
 
-    public async registerClient(
+    public async registerBackendClient(
```

Comment:
since we have separate paths for backend and frontend let's be explicit about it